tests/e2e/orchestrator/orchestrator.go

```diff
@@ -291,27 +291,39 @@ func (o *Orchestrator) runValidators(t *testing.T) {
 	rpcClient, err := rpchttp.New(rpcURL, "/websocket")
 	require.NoError(t, err)
 
-	require.Eventually(t,
-		func() bool {
-			ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
-			defer cancel()
-
-			status, err := rpcClient.Status(ctx)
-			if err != nil {
-				return false
-			}
+	checkHealth := func() bool {
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+		defer cancel()
 
-			// let the node produce a few blocks
-			if status.SyncInfo.CatchingUp || status.SyncInfo.LatestBlockHeight < 3 {
-				return false
-			}
+		status, err := rpcClient.Status(ctx)
+		if err != nil {
+			return false
+		}
 
-			return true
-		},
-		2*time.Minute,
-		time.Second,
-		"Ojo node failed to produce blocks",
-	)
+		// let the node produce a few blocks
+		if status.SyncInfo.CatchingUp || status.SyncInfo.LatestBlockHeight < 3 {
+			return false
+		}
+
+		return true
+	}
+
+	isHealthy := false
+	for i := 0; i < 40; i++ {
```

Comment:
Can we put the max wait time as a const here? something like
``` golang
maxHealthcheckSeconds := 40
```

Reply:
https://github.com/ojo-network/ojo/pull/132/commits/634b10eca1838c8172ea7b6eff3c8af8f8df08bd